### DISCUSSION TITLE
How to deploy Scala project by docker

### DISCUSSION THREAD

**Discussion by Liu-Eric-Yu** [playframework/playframework#11575] [Q&A]
### Problem I meet
I want to deploy my Scala project into docker.

When I dist project with  my MacBook(M2) by sbt  , and it successfuly disted.

And then I try to run it on my MacBook , It successed.

But I delpoy it to docker and try to run it. but it was pending with nothing to log.

I don`t know where the problem is and how to resolve it.

### My Local env
**Play Version** 
   play:2.8.18

**API**
scalaVersion : "2.13.10"

**sbt**
sbt.version: 1.8.0

### Operating System
MacBook Pro（13Inch , M2，2022）

### JDK
java version "11.0.17" 2022-10-18 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.17+10-LTS-269)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.17+10-LTS-269, mixed mode)

### Docker env
The docker env what I need was like this.
`brew install colima` 
`brew install docker` 
`colima start --memory 4 --arch x86_64` 
`docker run ...` 

### DockerFile
![image](https://user-images.githubusercontent.com/100890742/207821378-e7a45d29-17b2-451f-b74f-d45589003cb0.png)

### What I need
Can You tell me  , When I deploy Scala project to docker. What is the docker necessary Configuration.

Could you please give me some guidance。Thanks very much.

**Comment by PromanSEW** [comment]
https://www.scala-sbt.org/sbt-native-packager/formats/docker.html
Here is sample project how to configure Docker:
https://github.com/PromanSEW/ebean_example/blob/main/build.sbt

**Comment by Squeng** [comment]
Maybe my Dockerfile can be of use to you, too:
https://github.com/Squoss/Squawg/blob/main/Dockerfile
